index.js

```diff
@@ -200,6 +205,16 @@ if (config.credentials.aws.credential_file) {
         organization: config.credentials.github.organization,
         login: config.credentials.github.login
     };
+} else if (config.credentials.alibaba.credential_file && (!settings.cloud || (settings.cloud == 'alibaba'))) {
+    settings.cloud = 'alibaba';
+    cloudConfig = loadHelperFile(config.credentials.alibaba.credential_file);
+} else if (config.credentials.alibaba.token && (!settings.cloud || (settings.cloud == 'alibaba'))) {
+    settings.cloud = 'alibaba';
+    checkRequiredKeys(config.credentials.alibaba, ['access_key_secret']);
+    cloudConfig = {
+        accessKeyId: config.credentials.aws.access_key_id,
```

Comment:
Will this be config.credentials.aws.access_key_id or config.credentials.alibaba.access_key_id?